packages/api/test/e2e/promise-alex-archive.spec.ts

```diff
@@ -34,25 +37,23 @@ describe.skip('alex archive queries (local)', () => {
 
   // https://github.com/polkadot-js/api/issues/846
   it('handles toJSON with no issues', async (done) => {
-    const signedBlock = await api.rpc.chain.getBlock('0x85c62b581f38cb81c3e443d34392672beb1fb877017fd7237cc87704113259dc');
-    const failed = signedBlock.block.extrinsics.filter((extrinsic) => {
-      try {
-        const json = extrinsic.method.toJSON();
-
-        console.error(json);
-
-        return false;
-      } catch (error) {
-        console.log(extrinsic.method);
-        console.log(extrinsic.method.keys());
-        console.error(error);
-
-        return true;
-      }
-    });
-
-    expect(failed).toHaveLength(0);
-
-    done();
+    api.rpc.chain.getBlock('0x85c62b581f38cb81c3e443d34392672beb1fb877017fd7237cc87704113259dc', result => {
+      const failed: boolean = result.block.extrinsics.filter((extrinsic: Extrinsic) => {
```

Comment:
failed is not a boolean here, it's an array of extrinsics. If we manually cast something here, it's better to do it on `result` here

Reply:
Ok, I changed the type to `Array<Extrinsic>`.
Not sure what you mean with the second sentence, there's no `result` in the test.

Reply:
Well, actually, the test fails when I change the test to `Array<Extrinsic>` , it passes with `boolean`
<img width="984" alt="Screenshot 2019-05-24 at 12 18 38" src="https://user-images.githubusercontent.com/125398/58321315-1badbb80-7e1e-11e9-8695-ec6ac88b0c4b.png">

Reply:
I meant: `(result: Block)`

Reply:
Ah, ok, the test still fails, though.

Reply:
TS typings have no impact no tests, so the test is failing for some other reason.

In this file, I think you need to run an Alex archive node locally. We can `.skip` for now, if one day there's a public URL pointing to an archive node we put it back.

Reply:
Yes, I'm an idiot.  Of course we don't have access to a random old block on alex... Sorry things are starting to get blurry, I've spent too much time with tslint :)

Reply:
I added a comment and skipped the test for now.